DevEdu.API/DevEdu.Business/Services/CourseService.cs

```diff
@@ -13,6 +14,8 @@ public class CourseService : ICourseService
         private readonly ITopicRepository _topicRepository;
         private readonly ITaskRepository _taskRepository;
         private readonly IMaterialRepository _materialRepository;
+        private const string _validationExceptionMessageSamePositions = "the same positions of topics in the course";
+        private const string _validationExceptionMessageSameTopics = "the same topics in the course";
```

Comment:
close, but not enough)
the idea is to have only one constant per message. and you this constant in both test and service methods